src/Core/Grid/Definition/Factory/MetaGridDefinitionFactory.php

```diff
@@ -47,30 +47,6 @@
  */
 final class MetaGridDefinitionFactory extends AbstractGridDefinitionFactory
 {
-    /**
-     * @var string
-     */
-    private $resetActionUrl;
-
-    /**
-     * @var string
-     */
-    private $redirectionUrl;
-
-    /**
-     * MetaGridDefinitionFactory constructor.
-     *
-     * @param string $resetActionUrl
-     * @param string $redirectionUrl
-     */
-    public function __construct(
```

Comment:
⚠️ BC Break
... or is it ? What happens when too many arguments are given to a constructor ? For php5.6 I think it's fine, but I'm not sure about php7.1 or php7.2

Reply:
I think no one is ever using directly the `metaGridDefinitionFactory` - if someone is using then it is a service `prestashop.core.grid.definition.factory.meta` is used instead which causes no harm. And since the removed parameters were private it is totally safe to remove them :)